sdk/loadtestservice/azure-developer-loadtesting/swagger/README.md

```diff
@@ -13,6 +13,13 @@ package-pprint-name: Azure Developer LoadTesting
 security: AADToken
 security-scopes: https://cnt-prod.loadtesting.azure.com/.default
 directive:
+  - from: swagger-document
+    where: $["paths"]["/tests/{testId}/files/{fileName}"].put
+    transform: $["operationId"] = "LoadTestAdministration_BeginUploadTestFile";
+  - from: swagger-document
+    where: $["paths"]["/test-runs/{testRunId}"].patch
+    transform: $["operationId"] = "LoadTestRun_BeginTestRun";
```

Comment:
ideally you make this `$["x-ms-long-running-operation"] = true`, but this does technically work so won't force you to change it